commcare_connect/opportunity/views.py

```diff
@@ -1050,7 +1050,11 @@ def user_visit_review(request, org_slug, opp_id):
     opportunity = get_opportunity_or_404(opp_id, org_slug)
     if not opportunity.managed:
         return redirect("opportunity:detail", org_slug, opp_id)
-    is_program_manager = request.org_membership.is_admin and request.org.program_manager
+    is_program_manager = (
+        request.org_membership != None  # noqa: E711
+        and request.org_membership.is_admin
```

Comment:
why does it need to be an admin? I think in the past we talked about it being a superuser but im not sure we want to restrict it to only admins. Since most of these orgs will act exclusively as program managers, it seems wrong to block the members from acting as program managers

Reply:
Yeah, this is based off of the Program Spec doc that mentions "All features in the spec should be restricted to organization admins of organizations marked as “program_manager”". I think what you said above makes a lot more sense than what the document mentions.